terminal/termbox/cell_options_test.go

```diff
@@ -60,6 +60,8 @@ func TestCellFontModifier(t *testing.T) {
 		{cell.Options{Underline: true}, tbx.AttrUnderline, false},
 		{cell.Options{Italic: true}, 0, true},
 		{cell.Options{Strikethrough: true}, 0, true},
+		{cell.Options{Inverse: true}, tbx.AttrReverse, true},
```

Comment:
When we change `wantErr` to `false` to false, the tests seem to pass and are covering the new lines in `termbox/cell_options.go`.